docs/sources/enterprise/access-control/permissions.md

```diff
@@ -23,50 +23,50 @@ scope
+| `users:disable`            | `global:users:*`                                                                        | Disable a user.                                                                                                                                          |
+| `users.permissions:update` | `global:users:*`                                                                        | Update a user’s organization-level permissions.                                                                                                          |
+| `users:logout`             | `global:users:*`                                                                        | Log out a user.                                                                                                                                          |
+| `users.quotas:list`        | `global:users:*`                                                                        | List a user’s quotas.                                                                                                                                    |
+| `users.quotas:update`      | `global:users:*`                                                                        | Update a user’s quotas.                                                                                                                                  |
+| `org.users.read`           | `users:*`                                                                               | Get user profiles within an organization.                                                                                                                |
+| `org.users.add`            | `users:*`                                                                               | Add a user to an organization.                                                                                                                           |
+| `org.users.remove`         | `users:*`                                                                               | Remove a user from an organization.                                                                                                                      |
+| `org.users.role:update`    | `users:*`                                                                               | Update the organization role (`Viewer`, `Editor`, `Admin`) for an organization.                                                                          |
+| `ldap.user:read`           | n/a                                                                                     | Get a user via LDAP.                                                                                                                                     |
```

Comment:
I am wondering about this verb and see that the code uses "read" and the plain text uses "get". What is happening here? Asked another way, what is the point of getting/reading a user from LDAP?

Reply:
There is currently [another conversation](https://raintank-corp.slack.com/archives/C01LU2Y5527/p1631179555006300) about consistency and naming the roles/permissions . Since this is not something changed in this PR, perhaps we can move that discussion there.